src/d3a/models/area/__init__.py

```diff
@@ -414,6 +427,7 @@ def update_area_current_tick(self):
             child.update_area_current_tick()
 
     def tick_and_dispatch(self):
+        """Invokes tick handler and broadcast the event to children."""
```

Comment:
Nit: [PEP 257](https://www.python.org/dev/peps/pep-0257/) recommends using the imperative form for docstrings:
> The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".

So this would be 
```suggestion
        """Invoke tick handler and broadcast the event to children."""
```
Minor anyway.

Reply:
Thanks for this comment! I always wondered that and always did i wrong. Will change it from now on.